src/Cli/dotnet/commands/dotnet-test/TestingPlatformCommand.cs

```diff
@@ -210,6 +331,58 @@ private static bool TryGetUnknownMessage(IRequest request, out UnknownMessage un
             return false;
         }
 
+        private void OnHandshakeInfoReceived(object sender, HandshakeInfoArgs args)
+        {
+            if (VSTestTrace.TraceEnabled)
```

Comment:
Nit: feel free to ignore - all these methods could benefit from early return :) It's always good to avoid nesting when possible.